app/src/main/java/br/com/dcarv/arcmovies/Utils.kt

```diff
@@ -0,0 +1,10 @@
+package br.com.dcarv.arcmovies
+
+import android.view.View
+
+/**
+ * @author Danilo Carvalho
+ */
+fun View.show(show: Boolean) {
+    this.visibility = if (show) View.VISIBLE else View.GONE
+}
```

Comment:
I haven't really thought about this much until now and I haven't read at all about best practices regarding this, because I haven't been using extension functions that much.

I probably should have put it somewhere in the presentation package and called it ViewUtils.kt, since it's a platform specific function.